### PR TITLE
Naive 1.16.3+ port

### LINKED ISSUE #31: Minecraft 1.16.1 / Forge 32 support
It would be nice to have some (preview) support for Minecraft 1.16.1 using [Forge 32](https://files.minecraftforge.net/maven/net/minecraftforge/forge/index_1.16.1.html).

### LINKED ISSUE #34: 1.16.4 Forge support
Hello,

Me and my friend would love to use this mod on our server.

if you need anyone to test feel free to contact me.

### PR DESCRIPTION
**Pull request by dantaeusb**
Changed some class/method signatures to work on 1.16.3. Implementation was tested on my own Minecraft 1.16.4 and Mumble (MurMur) server, no issues found so far.

Something may have to be changed here but I guess it's a good place to start.

Should solve #31

### CODE REVIEW

**Comment by zsawyer** [comment]
Hello and thank you for your valuable contribution. I am still checking. This looks really good. Sometimes it's just that simple. 

PS: I think Vec3D has  been renamed 4 times now during this project.

**Comment by GoBoopADog** [comment]
I've been trying to compile this PR myself but haven't been able to figure out how to as I've never tried compiling my own mods before. Please accept the PR so I (And many others) can use it :)

**Comment by dantaeusb** [comment]
@GoBoopADog you can use the one I've built for myself in meantime https://www.icloud.com/iclouddrive/0pE5IqPiggqIGFFljUV8CX9XA#mumblelink-1.16.3-4.5.1 . Don't forget to enable the position tracking in mumble settings.

**Comment by Wearfoi** [comment]
> @GoBoopADog you can use the one I've built for myself in meantime https://www.icloud.com/iclouddrive/0pE5IqPiggqIGFFljUV8CX9XA#mumblelink-1.16.3-4.5.1 . Don't forget to enable the position tracking in mumble settings.

I'm having a problem with this mod. I've downloaded it to my mods folder on version 1.16.3 but Minecraft tells me I need javafml:35. How do I fix this?

File: mod/src/main/resources/META-INF/mods.toml [hunk 1]
```diff
@@ -6,9 +6,11 @@
 # The name of the mod loader type to load - for regular FML @Mod mods it should be javafml
 modLoader="javafml" #mandatory
 # A version range to match for said mod loader - for regular FML @Mod it will be the forge version
-loaderVersion="[25,)" #mandatory (24 is current forge version)
+loaderVersion="[35,)" #mandatory (24 is current forge version)
```
Comment (dantaeusb) [inline mod/src/main/resources/META-INF/mods.toml:9]: Could be `[34,)` as we're using mappings for 1.16.3.

**Comment by dantaeusb** [comment]
> I'm having a problem with this mod. I've downloaded it to my mods folder on version 1.16.3 but Minecraft tells me I need javafml:35. How do I fix this?

Sorry, I've added an extra version dependedncy and didn't build/check that for 1.16.3, even though it possible with fixing line above. Try to use Minecraft 1.16.4 with Forge 35 if you can, or build this mod with fixed Forge dependency in comment above.

**Comment by Wearfoi** [comment]
> > I'm having a problem with this mod. I've downloaded it to my mods folder on version 1.16.3 but Minecraft tells me I need javafml:35. How do I fix this?
> 
> Sorry, I've added an extra version dependedncy and didn't build/check that for 1.16.3, even though it possible with fixing line above. Try to use Minecraft 1.16.4 with Forge 35 if you can, or build this mod with fixed Forge dependency in comment above.

I dont know much about coding sorry, do you have an older build suited for 1.16.3?

**Comment by TheMinesweep3r** [comment]
Hi, I am having some trouble getting this mod to work with 1.16.4. it seems to link with mumble just fine, and some positional audio works where depending where you are relative to another player it will play in different ears, but I can hear the player from an infinite distance, and also can still hear them in other dimensions.

**Comment by aybri** [comment]
Will the .jar from iCloud work with 1.16.5?

**Comment by aybri** [comment]
Nevermind, it does.

**Comment by hackerstudios** [comment]
> @GoBoopADog you can use the one I've built for myself in meantime https://www.icloud.com/iclouddrive/0pE5IqPiggqIGFFljUV8CX9XA#mumblelink-1.16.3-4.5.1 . Don't forget to enable the position tracking in mumble settings.

Apple id sucks send something else PLEASEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Comment by hackerstudios** [comment]
> @GoBoopADog you can use the one I've built for myself in meantime https://www.icloud.com/iclouddrive/0pE5IqPiggqIGFFljUV8CX9XA#mumblelink-1.16.3-4.5.1 . Don't forget to enable the position tracking in mumble settings.

please send something else other than apple APPLE SUCKS  THEY LOCKED MY ACCOUNT send something downloadable for everyone without having to make an apple account

**Comment by dantaeusb** [comment]
> please send something else other than apple APPLE SUCKS THEY LOCKED MY ACCOUNT send something downloadable for everyone without having to make an apple account

Haha, those *** managed to vendor-lock cloud storage, omg. It was accessible with incogtito before, now it asks for Apple ID :) 

Here's same jar in Google Drive @hackerstudios: https://drive.google.com/file/d/1C-Lc_noZK64m8KJ53nDiY20V3_MZCqQt/view?usp=sharing

**Comment by zsawyer** [comment]
Thanks again for the PR - though this will not be merged directly this helped me massively for creating the 1.16.x releases.

**Comment by elkinsa** [comment]
Quick question about this. Since a version for 1.16.5 isn't available but I am wanting to use it on my server, what might I need to change to get this working for now? I am very familiar with Java (it is my primary language at work), but I am not familiar with Gradle or any specific things needed for Minecraft modding (minus obviously adding them to my game). I thought I had figured it out, but it just gives me an error in game.

**Pull request closed without merging**